### DISCUSSION TITLE
I created a fork and called it ZhitHead 2

### DISCUSSION THREAD

**Discussion by robreiss** [aod/zhithead#61] [General]
Thanks aod for the great project. I wanted to learn XState a bit and found your project. It was a great way to get introduced to the concepts in XState version 5. 

If anyone is interested the code is at [https://github.com/robreiss/zhithead2](https://github.com/robreiss/zhithead2) you can play it at https://zhithead2.vercel.app/

**Comment by aod** [comment]
Hi, I appreciate your interest a lot. The code is a bit old by now but great to hear it still could help you in learning about XState 🙌

Interesting that you've ported it to NextJS. I don't see the use case in that necessarily but I suppose it can be a fun exercise.
